Dockerfile

```diff
@@ -4,5 +4,5 @@ WORKDIR /usr/src/app
 COPY package*.json ./
 RUN npm install
 COPY . .
-EXPOSE 80
+EXPOSE 3000
```

Comment:
Why is this being changed to 3000?

Reply:
Instead of changing this to 3000, can you change the port exposed in the docker compose to 3000

Reply:
Actually I'm not too sure. Let's discuss in car ride to airport

Reply:
@theevenprime2 Can you change the `.env.local` APP_PORT variable to 3000?